docs/overview/first-steps.md

```diff
@@ -147,14 +145,16 @@ Below you can find some extra information about **[Providers](./providers.md)**,
 
 #### Providers
 
-Contain 2 providers:
+Contain 3 providers:
 
 - **application**: This provider is a class that extend from the Application class from @expressots/core that provides the application **[Life Cycle Hooks](application.md#lifecycle-hooks)**.
```

Comment:
Remove bold from application, bindingtype, db-in-memory, binding and db-in-memory